examples/OCPS.ipynb

```diff
@@ -0,0 +1,341 @@
+{
```

Comment:
<p>I didn't appreciate the OODS publishes one event per CCD which basically necessitates why you had to write this loop. Is there not an "entire image ready" type event? I think we'll want one.</p><p> </p><p>Also, I suggest adding a timeout in the <code>.next</code> area or you might be waiting for an eternity.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
It is a little tricky for any element of the system to maintain the state necessary to output an "entire image ready" event.  And what happens if (as is currently occurring), some CCDs are written/archived and others are not?  Do you return a partial image?  How long do you want to wait?